README.md

```diff
@@ -273,3 +273,24 @@ following ways, in precedence order:
   in the case where you specify a path to a Python script via the `APP_FILE` environment
   variable, defaulting to a file named `app.py` if it exists. The script is
   passed to a regular Python interpreter to launch your application.
+
+Hot deploy
+---------------------
+
+If you are using Django, hot deploy will work out of the box.
```

Comment:
Is the Gunicorn config file has set  `reload` option to true by default? And the `APP_CONFIG` env var is s pointing to the right config by default ?
If not nuke this line, cause thne the hot deploy is not working out of the box  
-